### PR TITLE
make sure the node is cleaned up after vm migration

### PR DESCRIPTION
**Pull request by john-kheir**

### CODE REVIEW

**Comment by john-kheir** [comment]
```
root@core0:/0-orchestrator/tests/0_orchestrator/test_suite# nosetests -v -s  testcases/basic_tests/test07_vms_apis.py:TestVmsAPI.test011_post_nodes_vms_vmid_migrate  --tc-file=config.ini
GAT-077 ... ok

----------------------------------------------------------------------
Ran 1 test in 180.140s

OK
```

File: tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py [hunk 1]
```diff
@@ -415,6 +416,13 @@ def test011_post_nodes_vms_vmid_migrate(self):
         vms = new_core0_client.client.kvm.list()
         self.assertIn(self.data['id'], [x['name'] for x in vms])
 
+        self.lg.info('check that the node, VM0 moved to, is cleaned up')
+        res = new_core0_client.client.bash('ps -a | grep "sshd.config"').get()
+        self.assertNotIn('myvm', res.stdout)
```
Comment (ahmedelsayed-93) [inline tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py]: myvm should be the vm id ```self.data['id']```

File: tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py [hunk 2]
```diff
@@ -415,6 +416,13 @@ def test011_post_nodes_vms_vmid_migrate(self):
...
+        res = new_core0_client.client.bash('ls /tmp').get()
+        self.assertNotIn('myvm', res.stdout)
```
Comment (ahmedelsayed-93) [inline tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py]: myvm should be the vm id (self.data['id'])

File: tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py (hunk 1 above)
Comment (john-kheir) [reply tests/0_orchestrator/test_suite/testcases/basic_tests/test07_vms_apis.py]: Done

**Pull request merged**